tests/test_unit.py

```diff
@@ -162,8 +142,7 @@ def runTest(self):
         finally:
             pp.ParserElement.setDefaultWhitespaceChars(prev_default_whitespace_chars)
 
-class UpdateDefaultWhitespace2Test(ParseTestCase):
-    def runTest(self):
+    def testUpdateDefaultWhitespace2(self):
```

Comment:
All test classes are converted to test methods, and renamed to start with "test"